src/azul/plugins/metadata/anvil/__init__.py

```diff
@@ -285,6 +288,9 @@ def manifest_config(self) -> ManifestConfig:
         # the fields listed here and those used in `self._field_mapping`.
         fields_to_omit_from_manifest = [
             ('contents', 'activities', 'activity_table'),
+            # duos_id is omitted from manifests since the field isn't included
+            # in the `files` index.
```

Comment:
This isn't very informative. The reason is that there is only one DUOS bundle per dataset and that that bundle only contributes to outer entities of the `datasets` type, and not to entities of the other types, such as files, which the manifest is generated from.

This comment should also appear in the test assertion, the one that prompted my review comment, as we discussed.